src/AccessLog.php

```diff
@@ -115,27 +104,45 @@ public function process(ServerRequestInterface $request, DelegateInterface $dele
     }
 
     /**
+     * Generates and returns the message format
+     *
+     * @return string
+     */
+    private function getFormat()
+    {
+        $message = $this->vhost ? '%v' : '%h';
```

Comment:
the example in https://httpd.apache.org/docs/2.4/logs.html#accesslog (Section "Virtual Hosts") contains the following format:
```
LogFormat "%v %l %u %t \"%r\" %>s %b" comonvhost
```